package.json

```diff
@@ -6,10 +6,11 @@
     "bootstrap": "lerna bootstrap --no-ci",
     "test": "lerna run test",
     "build": "lerna run build",
-    "release": "lerna version",
+    "release": "del-cli packages/**/package-lock.json && lerna version",
```

Comment:
can't we just `rm packages/**/package-lock.json`?

Reply:
This was my initial idea but `del-cli` is cross platform :)